buildSrc/src/main/groovy/org/elasticsearch/gradle/test/ClusterConfiguration.groovy

```diff
@@ -137,7 +137,7 @@ class ClusterConfiguration {
         this.project = project
     }
 
-    Map<String, String> systemProperties = new HashMap<>()
+    Map<String, Object> systemProperties = new HashMap<>()
```

Comment:
The difference is easy to overlook , maybe it would be good to have a comment explaining why `Object` is needed. 
We might be able to have `Map<String, GString>` to make the intention cleaner.

Reply:
💯